frontend/src/components/object/ObjectVersion.vue

```diff
@@ -39,15 +40,18 @@ const { getUserSearch } = storeToRefs(userStore);
 const tableData: Ref<Array<VersionDataSource>> = ref([]);
 
 // Actions
+const router = useRouter();
 const toast = useToast();
 
-// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-const showInfo = async (id: string) => {
-  // emit('show-object-info', id);
-};
-
-function onDeletedSuccess() {
+async function onDeletedSuccess() {
   toast.success('File deleted');
+  await versionStore.fetchVersions({ objectId: props.objectId });
```

Comment:
i think i found when deleting any version except latest, that version remains in the table until you do a hard refresh.
maybe the fetchVersions() needs to do that partition logic somewhere to remove that old version from the store (?)

Reply:
or maybe it's because the route isnt changing 🤔